### PR TITLE
created threads, added logger

### PR DESCRIPTION
**Pull request by AndriiBardin**

### CODE REVIEW

**Review by AndreiYazadzhi — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.idea/*
+*.iml
+target/*
```
Comment (RomanDubovskyi) [inline .gitignore]: empty line at the end of the file

File: log/log.log [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+ 12:14:43,440 INFO  CustomThread:14 - Thread 91
+ 12:14:43,440 INFO  CustomThread:14 - Runnable 92
+ 12:14:43,440 INFO  CustomThread:14 - Thread 93
+ 12:14:43,440 INFO  CustomThread:14 - Runnable 94
+ 12:14:43,441 INFO  CustomThread:14 - Thread 95
+ 12:14:43,441 INFO  CustomThread:14 - Runnable 96
+ 12:14:43,441 INFO  CustomThread:14 - Thread 97
+ 12:14:43,441 INFO  CustomThread:14 - Runnable 98
+ 12:14:43,441 INFO  CustomThread:14 - Thread 99
+
```
Comment (RomanDubovskyi) [inline log/log.log:201]: same

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+                    <configuration>
+                        <source>${jdk.version}</source>
+                        <target>${jdk.version}</target>
+                        <encoding>${project.build.sourceEncoding}</encoding>
+                    </configuration>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+    </build>
+</project>
```
Comment (RomanDubovskyi) [inline pom.xml]: same here

File: src/main/java/CustomRunnable.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import org.apache.log4j.Logger;
+
+public class CustomRunnable implements Runnable {
+    private static final Logger logger = Logger.getLogger(CustomThread.class);
```
Comment (RomanDubovskyi) [inline src/main/java/CustomRunnable.java:4]: wrong class for logger

File: src/main/java/Main.java [hunk 5]
```diff
@@ -0,0 +1,9 @@
+public class Main {
+    public static void main(String[] args) {
+        Counter counter = new Counter();
+        CustomRunnable runnable = new CustomRunnable(counter);
+        CustomThread myThread = new CustomThread(counter);
```
Comment (RomanDubovskyi) [inline src/main/java/Main.java]:
```suggestion
        Runnable runnable = new CustomRunnable(counter);
        Thread myThread = new CustomThread(counter);
```

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**